helm-charts/common/teirerank/README.md

```diff
@@ -21,6 +21,35 @@ MODELDIR=/mnt/opea-models
 
 MODELNAME="/data/BAAI/bge-reranker-base"
 
+## HorizontalPodAutoscaler (HPA) support
+
+`horizontalPodAutoscaler` option enables HPA scaling for the deployment:
+https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+
+Autoscaling is based on custom application metrics provided through [Prometheus](https://prometheus.io/).
+
+### Pre-conditions
+
+If cluster does not run [Prometheus operator](https://github.com/prometheus-operator/kube-prometheus)
+yet, it SHOULD be be installed before enabling HPA, e.g. by using:
+https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack
+
```

Comment:
```suggestion

`horizontalPodAutoscaler` enabled in top level Helm chart depending on this component  (e.g. `chatqna`), so that relevant custom metric queries are configured for PrometheusAdapter.

```